This patch avoids rungms from failing in cases where PPN (processors per node)
is not defined.

Written by Oliver Stueker, ACENET/Compute Canada
Memorial University of Newfoundland, St. John's, Canada
--- rungms.orig	2017-07-19 16:53:29.325530604 +0000
+++ rungms	2017-07-19 16:55:39.627358245 +0000
@@ -538,6 +538,7 @@
    #      all nodes are presumed to have equal numbers of cores.
    #
    set PPN=$4
+   if (null$PPN == null) set PPN=1 # make sure PPN is initialized
    #
    #      Allow for compute process and data servers (one pair per core)
    #      note that NCPUS = #cores, and NPROCS = #MPI processes
